cmd/neofs-node/cache.go

```diff
@@ -319,3 +319,32 @@ func (s *ttlContainerLister) InvalidateContainerListByCID(id *cid.ID) {
 		}
 	}
 }
+
+type cachedIRFetcher ttlNetCache
+
+func newCachedIRFetcher(f interface{ InnerRingKeys() ([][]byte, error) }) *cachedIRFetcher {
+	const (
+		irFetcherCacheSize = 1 // we intend to store only one value
+		irFetcherCacheTTL  = 30 * time.Second
```

Comment:
Can we invalidate the cache on some event or add a comment on why 30-sec is not so bad (IR changes not so often)?

Reply:
Added.